namespaces/ci_job.yml

```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
     type: integer
     description: >
       Build ID of the CI job.
 
-  - name: build_timestamp
+  - name: timestamp
```

Comment:
I know this is not a new field, but how is this field different than the top level `@timestamp` field?  Can we use the top level `@timestamp` field instead and get rid of this field?